scrapers/Mormongirlz.yml

```diff
@@ -0,0 +1,32 @@
+name: mormongirlz
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - mormongirlz.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="h-contant"]/h2/text()
+      Date:
+        selector: //meta[@property="article:published_time"]/@content
+        postProcess:
+         - replace: 
+              - regex: (.+)(T.+)
+                with: $1
+      Details: //div[@class="h-contant"]/div/text()
```

Comment:
details gets only part of the description in scenes that have line breaks in the text (eg `https://mormongirlz.com/pearl-the-unveiling` and `https://mormongirlz.com/dolly-the-covenant`)
Using either 
```yaml
      Details: //div[@class="h-contant"]/div
```
or 
```yaml
      Details: 
        selector: //div[@class="h-contant"]/div/text()
        concat: "\n\n"
```
gets the whole description text 
The second part maintains the newlines in the text